satisfactory-accounting-app/src/node_display/group/group_name.rs

```diff
@@ -124,13 +135,21 @@ impl GroupName {
         let oninput = link.callback(|input| Msg::UpdatePending {
             pending: get_value_from_input_event(input),
         });
+        let onkeydown = link.callback(|e: KeyboardEvent| {
```

Comment:
You can use `batch_callback` for optional callbacks -- it accepts both Vec<Msg> and Option<Msg>, with None just resulting in no callback.

https://docs.rs/yew/latest/yew/html/struct.Scope.html#method.batch_callback

Reply:
Fixed.